### ISSUE TITLE
Create the service layer

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#27]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To use services in controllers, served through DI.

### Why

Testability, reusability.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #26
